data-processing/data_manipulation/common/config.py

```diff
@@ -70,23 +76,23 @@ def __set_property_value(self, yaml_data):
             parent_key='minio',
             key='access_key',
             env_name='MINIO_ACCESSKEY',
-            default_value='hpU4SCmj5jiU7IP5'
+            default_value='xxxxxx'
```

Comment:
don't use xxxxx as it's meaningless, we can use something like 'faked_minio_ak', 'faked_minio_sk', 'faked_minio_api_url', so we can know what's the wrong value if any issue occurs.

Reply:
ok